spring-integration-file/src/main/java/org/springframework/integration/file/FileWritingMessageHandler.java

```diff
@@ -79,10 +89,17 @@
  * @author Gary Russell
  * @author Tony Falabella
  */
-public class FileWritingMessageHandler extends AbstractReplyProducingMessageHandler {
+public class FileWritingMessageHandler extends AbstractReplyProducingMessageHandler
```

Comment:
I think we need right class level JavaDocs as well to reflect the new `flush` logic.